src/main/java/com/instana/operator/resource/KubernetesResource.java

```diff
@@ -7,7 +7,7 @@
 
 import java.util.function.Predicate;
 
-public class ResourceUtils {
+public class KubernetesResource {
```

Comment:
I think `Util` or `Helper` has a place for utility methods that are unattached to a particular class. This seems to indicate it declares a "thing" that is a `KubernetesResource`, not helper methods to work them. JMO.

Reply:
I'm not particularly a fan of the `util` as it becomes a dumping ground. This class could be split into pod and metadata.

Reply:
It could be if left unchecked and used indiscriminately. But the alternative is to not indicate it's just a holder class for helper methods. Not sure that's a net gain IMO.

Reply:
Might be as simple as using the plural form rather than singular. `KubernetesResources` seems better to me than just the singular form, which implies something else in my mind.

Reply:
Will let it ruminate in the back of my head for now.